client-java/controller/src/main/java/org/evomaster/client/java/controller/problem/GraphqlProblem.java

```diff
@@ -0,0 +1,18 @@
+package org.evomaster.client.java.controller.problem;
+
+
+public class GraphqlProblem implements ProblemInfo{
+
+    public String graphqlEndpointUrl;
```

Comment:
only in DTOs mutable fields should be `public`.
this should be `private` and also `final`. see for example `RestProblem`